src/router/Router.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import NavBar from "../components/NavBar";
+import Information from "../pages/Information";
+import Profile from "../pages/Profile";
+import FAQ from "../pages/FAQ";
+import Registration from "../pages/Registration";
+
+const Router = () => (
+  <BrowserRouter basename="/HacktoberFest">
+    <div className="App">
```

Comment:
nit: if we want to be able to style this navbar even more we should add NavBar to the className.

Reply:
Made this change.  You can see this here:  ff04b35d77ede341f74b9ca923718e9adefff20d